admin/class-admin-asset-manager.php

```diff
@@ -634,6 +634,10 @@ protected function styles_to_be_registered() {
 				'name' => 'inside-editor',
 				'src'  => 'inside-editor-' . $flat_version,
 			],
+			[
+				'name' => 'editor',
```

Comment:
I am thinking this will be included in more as the UI library + Tailwind stylesheet. But this is the first reason to have that, and might be premature

Reply:
Maybe I should rename it to `ui-library`? That way it is specific and we should never add to it, but rely on it via WP dependency for other styles? 🤔

Reply:
Ok, that is not how it works, I'm leaving it as is